.env.example

```diff
@@ -30,6 +30,7 @@ ELASTICSEARCH_URL="https://testnet-index.multiversx.com"
 MX_API_URL="https://testnet-api.multiversx.com"
 MX_GATEWAY_URL="https://testnet-gateway.multiversx.com"
 MX_DEX_URL="https://devnet-graphql-next.maiar.exchange"
+ELRONDDATAAPI_URL="https://testnet-tools.elrond.com/data-api/graphql"
```

Comment:
we should rename it to `MX_DATA_API_URL`

Reply:
will remove it on a later PR